starter-code/src/functions-and-arrays.js

```diff
@@ -59,6 +141,16 @@ var wordsFind = [
   'truth',
   'disobedience'
 ];
+function doesWordExist (myArray,word) {
+  var verif=false;
+  for (var i=0; i<myArray.length; i++) {
+    if (myArray[i] === word) {
+      verif=true;
+      break;
+    }
+  }
+  return verif;
+}
```

Comment:
the code works but it is not ideal to use a break statement, 'break' should only be used in very specific cases and if you need to use it, it usually means that there is a better way to write the code, 

``` javascript
function doesWordExist (myArray,word) {
  
  for (var i=0; i<myArray.length; i++) {
    if (myArray[i] === word) {
      return true;
    }
  }
  return false;
}
```
whenever a function returns something, the rest of the code inside of the function is not executed anymore. If the condition myArray[i] === word is verified, you got what you were looking for and you can return. If the condition if never verified,  you can return false at the very end of the function, after the loop.